qt/scientific_interfaces/General/CMakeLists.txt

```diff
@@ -1,20 +1,20 @@
-set(SRC_FILES MantidEV.cpp MantidEVWorker.cpp SampleTransmission.cpp StepScan.cpp)
+set(SRC_FILES FitScriptGenerator.cpp StepScan.cpp)
 
 # Include files aren't required, but this makes them appear in Visual Studio IMPORTANT: Include files are required in
 # the MOC_FILES set. Scroll down to find it.
-set(INC_FILES DllConfig.h MantidEV.h MantidEVWorker.h SampleTransmission.h StepScan.h Updateable.h)
```

Comment:
I notice `StepScan.h` has been removed here but not for the `SRC_FILES`. Is this intentional?

`FitScriptGenerator.h` also isn't here. Is the comment about the files not appearing in VS no longer the case?

Reply:
They do still appear in VS, so this comment must be outdated. I have updated the comment to make it clearer that only non-Qt header files need to be specified in the `INC_FILES` variable